flashinfer/comm/mixed_comm.py

```
@@ -0,0 +1,1205 @@
+                MixedCommMode.FUSED_OPT_BYTES2_UC,
+            ]
+        valid_mode_list.append(MixedCommMode.NCCL_ONE)
+        if self.para_info.use_mixed:
+            valid_mode_list.append(MixedCommMode.NCCL_TP_DP)
+        if self.use_autotune:
+            valid_mode_list.append(MixedCommMode.AUTOTUNE)
+        return valid_mode_list
+
+    def init_virtual_memory(self):
+        def get_socket_path(pid):
+            return f"/tmp/{pid}"
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)